src/RTUStatistics.hpp

```diff
@@ -0,0 +1,31 @@
+#ifndef MODBUS_RTUSTATISTICS_HPP
+#define MODBUS_RTUSTATISTICS_HPP
+
+#include <base/Time.hpp>
+#include <stdexcept>
+#include <string>
+
+namespace modbus {
+    struct RTUStatistics {
+        // Total InvalidCRC error count
+        int total_crc_error_count = 0;
+
+        // Total UnexpectedReply error count
+        int total_unexpected_reply_error_count = 0;
+
+        // Total Success count
+        int total_sucess_count = 0;
```

Comment:
use fixed-size integer types (e.g. uintXX_t) especially on types that will be put on orogen interfaces.